doc/lua-filters.md

```diff
@@ -2014,6 +2016,109 @@ Fields:
 :   track changes setting for docx; one of `accept-changes`,
     `reject-changes`, and `all-changes` (string)
 
+## WriterOptions {#type-writeroptions}
+
+Pandoc writer options
+
+Fields:
+
+`cite_method`
+:   How to print cites (JSON-like)
```

Comment:
I don't know how intelligible "JSON-like" is going to be to the reader.
Maybe it would be worth adding a note at the beginning that "JSON-like" means that the value will be the same as in pandoc's JSON serialization of WriterOptions.  But even that has the drawback that most users won't know what that serialization looks like, and won't know how to find out.  (The cli tool will never generate this.)  Maybe, then, we should just list the legal values here?

Reply:
OK, done. I think this warants at least some basic tests, so docs and code won't fall out of sync as easily. I'll add some basic checks.